applications/zDebugger/zDebugger.js

```diff
@@ -0,0 +1,105 @@
+//
+//  zDebugger.js
+//
+//  Created by Zetaphor, September 21st 2023.
+//  Copyright 2023 Overte e.V.
+//
+//  This application let you setup different Anti-Aliasing setup for HMD and Desktop.
+//  And it switches the configuration when the mode changes.
```

Comment:
I think you accidentally copied description too :)
Can you add copyright header in HTML file too?

Reply:
🤦 That's what I get for speedrunning a commit